src/cgame/cg_ents.cpp

```diff
@@ -1373,6 +1373,18 @@ void CG_AddPacketEntities()
 			}
 		}
 	}
+
+	static Cvar::Range<Cvar::Cvar<float>>
```

Comment:
Usually, CVars are extern, not static. They're also defined and declared outside of functions.
Any reason to change that convention?

Reply:
Only real reason is build time

Reply:
Yes it is uncommon in gamelogic. Only 3 Cvars from thse are not declared at top of file. Duplicated code is also common, yet it's considered a bad thing.

You need the '-n' parameter to grep to know where the cvars are put in the file, -c is only count occurrences.

Reply:
Just moved it at the beginning of the file.